src/content/docs/en/reference/integrations-reference.mdx

```diff
@@ -1190,6 +1190,18 @@ declare global {
 
 Astro reserves the `astro:` prefix for future built-in hooks. Please choose a different prefix when naming your custom hook.
 
+## Astro vite environments
+
+Since v6, Astro is based on `vite` v7 and it heavily uses its [environment APIs](https://vite.dev/guide/api-environment). 
+  
+Astro build environments:
+- `ssr`: the classic and default `ssr` environment of `vite`. Astro uses this environment to build dynamic routes.
+- `client`: the classic and default `client` environment of `vite`.  Astro uses this environment to generate client assets, such as scripts, styles, etc.
```

Comment:
Nit: Calling this and `ssr` "classic" is a little confusing to me. I think maybe instead you are wanting to say that they are vite-defined or built-in or something to that effect.